test/e2e/features/pageObjects/workAllocation/myWorkPage.js

```diff
@@ -297,6 +299,25 @@ class MyWorkPage extends TaskList {
     }
 
 
+    async isWorkFilterOfTypeDisplayed(filterType){
+
+        const filetTypeNormalized = filterType.toLowerCase().split(' ').join(''); 
```

Comment:
Is this supposed to be filterTypeNormalized?

Reply:
fixed typo